go-runtime/compile/parser.go

```diff
@@ -24,27 +23,25 @@ type directiveWrapper struct {
 //sumtype:decl
 type directive interface{ directive() }
 
-type directiveExport struct {
-	Pos lexer.Position
+type directiveVisibility struct {
+	Pos schema.Position `parser:"" protobuf:"1,optional"`
 
-	Export bool `parser:"@'export'"`
+	Visibility schema.Visibility             `parser:"@('public' | 'internal' | 'private')"`
```

Comment:
Can we somehow catch `//ftl:export` (probably not here) and return an error indicating to use these now?